skipper-mixin/Makefile

```diff
@@ -0,0 +1,23 @@
+JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
```

Comment:
from skipper's perspective, this is not a skipper mixin but rather a grafana+prometheus mixin, so calling this directory 'skipper-mixin' doesn't make the most sense. How about the following directory structure:

```
monitoring
|-> mixin
```

alternatively, 'mixin' can be also 'grafana-mixin' or 'prometheus-mixin', or in worst case 'grafana-prometheus-mixin', not sure which one is better. But having a generic 'monitoring' can be useful in the future for other non-Go packages. Can you please create a top-level 'monitoring' dir, and move the mixin code there?